storybook/pages/CreateChannelPopupPage.qml

```diff
@@ -19,15 +19,105 @@ SplitView {
 
         PopupBackground {
             anchors.fill: parent
+
+            Button {
+                anchors.centerIn: parent
+                text: "Reopen"
+
+                onClicked: dialog.open()
+            }
         }
 
         CreateChannelPopup {
```

Comment:
Yeah, I initially planned to have it fully working in storybook too/first but then realized it would basically mean reimplementing the whole logic and models from NIM here in JS :/